xtable-api/src/test/java/org/apache/xtable/model/metadata/TestTableSyncMetadata.java

```diff
@@ -56,7 +59,24 @@ private static Stream<Arguments> provideMetadataAndJson() {
             "{\"lastInstantSynced\":\"2020-07-04T10:15:30Z\",\"instantsToConsiderForNextSync\":[],\"version\":0}"),
         Arguments.of(
             TableSyncMetadata.of(Instant.parse("2020-07-04T10:15:30.00Z"), null),
-            "{\"lastInstantSynced\":\"2020-07-04T10:15:30Z\",\"version\":0}"));
+            "{\"lastInstantSynced\":\"2020-07-04T10:15:30Z\",\"version\":0}"),
+        // New version of metadata and JSON where two new fields are added
```

Comment:
"new" will be relative in the future so we should mention the names of the fields added